### PR TITLE
Terraformed ECS task for weekly report

### PR DESCRIPTION
**Pull request by hhm970**
- Configured ECS task in `report_ecs.tf`
- Edited `report_eventbridge.tf` to target ECS task definition
- Got rid of comments in `pipeline_lambdas.tf`

### CODE REVIEW

File: terraform/report_ecs.tf [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+resource "aws_ecs_task_definition" "weekly-report-task-def" {
+  family                = "c10-games-weekly-report-tf"
+  container_definitions = jsonencode([
+    {
+      name         = "games-weekly-report"
+      image        = "129033205317.dkr.ecr.eu-west-2.amazonaws.com/c10-games-weekly-report"
+      essential    = true
+      "environment": [
+                {
+                    "name": "AWS_KEY",
```
Comment (Eibironke) [inline terraform/report_ecs.tf:23]: nice

File: terraform/report_ecs.tf [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                {
+                    "name": "DB_PASSWORD",
+                    "value": var.DB_PASSWORD
+                }
+      ]
+    }
+  ])
+  requires_compatibilities = ["FARGATE"]
+  network_mode             = "awsvpc"
+  memory                   = 3072
```
Comment (Eibironke) [inline terraform/report_ecs.tf:55]: Could use less memory? Less expensive

**Review by Eibironke — APPROVED** [review]
Still good to go 👍

File: terraform/report_ecs.tf (hunk 2 above)
Comment (hhm970) [reply terraform/report_ecs.tf:55]: PDF take up a lot of memory; this is a mitigation measure, in case the programme crashes due to insufficient memory

**Pull request merged**